Chap_API_Proc_Mgmt.tex

```diff
@@ -468,7 +484,7 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_SPAWN_TOOL}{"pmix.spwn.tool"}{bool}{
-Indicate that the job being spawned is a tool.
+Indicate that the job being spawned is a tool.  The repercussions of setting this attribute varies based on implementation.  For example, some implementations may not perform cpu-binding on process marked as a tool. 
```

Comment:
PMIx implementation isn't doing anything here - we just pass the attribute informing the host that the spawn request is launching a tool. What that means is totally up to the host.

Reply:
Thanks for the feedback.  We will re-visit these and consider updating to make it clearer that PMIx is just the messenger in all of these use cases.   We just wanted to make it clear that the format or input/output was not designated by PMIx, but agree that the best way to do that is to make it cleat that PMIx is only passing these through.

Reply:
I think the question here again is that you stated the behavior "...varies based on implementation", but not on which implementation - the host or the PMIx library? I'm not aware of any PMIx repercussions, nor was that the intent of the attribute - it was provided so that the host environment would know this is a tool being spawned, and _if_ they wanted to do something different for tools, then they could do so.